### ISSUE TITLE
Add windows package

### ISSUE THREAD

**Issue by emanuele-f** [ntop/n2n#79]
Providing pre-built executables to download will be nice. I'd avoid making an installer since the only requirement is to install the tap driver (a simple link can be enough), whereas the AES support can be provided with a static OpenSSL linkage (see https://github.com/ntop/n2n/blob/dev/doc/Windows.md)

**Comment by lucktu** [comment]
Very well, thank you very much!

**Comment by Teteros** [comment]
Regarding _deploying_ n2n on Windows once it's on e.g Github releases.

I've tested [n2n-gui](https://sourceforge.net/projects/n2nedgegui) and can confirm it still works if you replace `edge` binary with one build from latest `dev`.
There seems to be several [other projects](https://github.com/search?q=n2n+gui) as well.

It's a very good idea to use the newest NDIS6 tap-windows driver in the package as at least the one bundled in sourceforge version is the old XP one, likely buggy or poorly supported on newer Windows 10 versions.

Builds for tap-windows are on openvpn's server:
http://build.openvpn.net/downloads/releases

Sadly the right driver that installs is a bit annoying to get as the drivers in .zip archives are not fully signed, only the installers have the newest signatures.
e.g At time of writing that would be: `tap-windows-9.23.3-I601.exe`

Whether the windows package has UI or not, I'd suggest using chocolatey as [tap-windows](https://chocolatey.org/packages/tapwindows) and dependent libs like openssl and available so n2n could just depend on them.

By chocolatey's conventions the command-line version might be `n2n.portable` and installable/gui version `n2n.install`.

**Comment by emanuele-f** [comment]
Please note that some instructions for windows are already present here: https://github.com/ntop/n2n/blob/dev/doc/Windows.md

**Comment by tlsalex** [comment]
Any update for windows package?

**Comment by ndo360** [comment]
Yes I'd love to have a windows package so I can tinker with the latest version of n2n. :)

**Comment by doutv** [comment]
Looking forward to a windows release package.
